public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -38,6 +38,12 @@ export default class CloudWatchDatasource {
       item.metricName = this.templateSrv.replace(item.metricName, options.scopedVars);
       item.dimensions = this.convertDimensionFormat(item.dimensions, options.scopedVars);
       item.period = String(this.getPeriod(item, options)); // use string format for period in graph query, and alerting
+      let hasInvalidStatistics = item.statistics.some(s => {
+        return s.indexOf('p') === 0 && !/p\d{2}\.\d{2}/.test(s);
```

Comment:
a comment for what this does would be great (and maybe a test), what is an invalid statistic?